src/toffy/rosetta.py

```diff
@@ -677,14 +677,16 @@ def rescale_raw_imgs(img_out_dir, scale=200):
         fov_dir = os.path.join(img_out_dir, fov)
         # create subdirectory for the new images
         sub_dir = os.path.join(fov_dir, "rescaled")
-        os.makedirs(sub_dir)
+        if not os.path.exists(sub_dir):
+            os.makedirs(sub_dir)
         chans = io_utils.list_files(fov_dir)
         # rescale each channel image
         for chan in chans:
-            img = io.imread(os.path.join(fov_dir, chan))
-            img = (img / scale).astype("float32")
             fname = os.path.join(sub_dir, chan)
-            image_utils.save_image(fname, img)
+            if not os.path.exists(fname):
```

Comment:
Per #310 looks like the solution @HPiyadasa wanted writes the `.tiff.` regardless of if a channel file already exists. This makes sense because if the user specifies an existing `sub_dir`, most likely they want to overwrite the existing data there.

Reply:
If you look at his code, the solution only writes anything if the "rescaled" subdir doesn't exist in the FOV folder. All reading and writing is done under the statement (`if not os.path.exists(subdir)`), so it's not possible to overwrite.

It really depends on the functionality that makes the most sense for rosetta, but I can't think of what could change about the extracted images that would cause a user to need to re-scale.

Reply:
I basically just added a sanity check to that verifies all the channels are actually in the subdir, rather than skipping that FOV entirely.